### PR TITLE
update tweetbot for deployment

### PR DESCRIPTION
**Pull request by jnewmano**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +0,0 @@
-FROM golang:alpine
```
Comment (Soypete) [inline Dockerfile:1]: why are you killing the docker file?

File: db/cockroach.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	fn := "cockroach-public-root.crt"
+	_, err := os.Stat(fn)
+	if err == nil {
+		return fn, nil
+	}
+	// assume we need to get the file
+
+	log.Println("loading cluster root certificate")
+
+	clusterID := os.Getenv("DB_CLUSTER_ID")
```
Comment (Soypete) [inline db/cockroach.go:23]: I like this alot... I definitely didn't think about it at the time

File: db/connection.go [hunk 3]
```diff
@@ -18,25 +18,39 @@ type Connection struct {
 func Connect(ctx context.Context) (*Connection, error) {
 	log.Println("Connecting to database...")
 
-	urlParams := "sslmode=verify-full&options=--cluster%3Dlanky-bird-5343"
+	rootCertPath, err := loadCockroachRootCert(ctx)
```
Comment (Soypete) [inline db/connection.go:21]: 👍🏻

File: run [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (Soypete) [inline run:1]: In your opinion what is the advantage of having a run script over a docker file?

Comment (jnewmano) [reply run:1]: This run script is only used when developing locally, so it is in charge of setting up and configuring the local development environment. A docker file typically won't have any environment information in it, so the environment configuration has to come from somewhere to facilitate local development.

**Review by Soypete — APPROVED** [review]

**Pull request merged**